tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -197,8 +197,10 @@ def parameters(cls):
                 optional=True, is_directory=True, description="TF Object Detection API Config."
             ),
             '_tf_custom_op_config_dir': PathField(
-                optional=True, is_directory=True, description="TF Custom Operation Config."
+                optional=True, is_directory=True, description="TF Custom Operation Config prefix."
             ),
+            '_transformations_config_dir': PathField(
+                optional=True, is_directory=True, description="Transformation config for Model Optimizer"),
```

Comment:
Shouldn't the description say it's a prefix, like it does for `_tf_custom_op_config_dir`?

Reply:
okey, added